docs/guides/rookieguide.md

```diff
@@ -1,7 +1,126 @@
+
+UWAGA!!! TSA I ARMĘ URUCHAMIAMY BEZ UPRAWNIEŃ ADMINA, BO NAM SIĘ ACRE WYSYPIE. Objawy wysypania to to, że plugin ACRE na TSie mamy niby odpalony, ale słyszymy wszystkich normalnie na TSie, nie przestrzennie i bez potrzeby wbijania na serwer.
+Jak już gra się uruchomi i pokaże nam główne menu, klikamy na opcję Join.
+
+![To takie proste!](../_data/guides/rookie/rookie3.png/)
+
+Nastąpi połączenie z serwerem. Hasło powinno wejść samo, ale w razie czego jest ono na wspomianym przed chwilą kanale z informacjami dla rekrutów.
+
+
+
+
+
+
+
+
+
+
+
+
+
+## **FAQ.**
+Q: Dlaczego w zapisach nie ma podanej modlisty/nie ma jej na stronie z modsetami?
+
+A: Najpewniej oznacza to, że do misji jest dużo czasu i autor nie przygotwał jeszcze modlisty, sprawdź ponownie bliżej terminu misji.
+
+Q: Misja jest za 5 minut, dlaczego i tak nie ma tego do pobrania?
+
+A: 
+Wyślij zapytanie do swojego opiekuna na discordzie lub wejdź na TS na kanał “Jestem nowy i czegoś nie wiem” (duża szansa że ktoś po chwili wejdzie i spróbuje ci pomóc). Poza tym w większości przypadków te mody potrafią sporo ważyć, szczególnie jak nigdy żadnego nie pobraliście wcześniej, więc sugeruję wcześniejsze ogarnianie tych rzeczy, przynajmniej na kilka godzin przed misją. 
+
+Q: Coś się popsuło i nie było mnie słychać (a jestem w grze), ACRE sadzi błędy
+
+A: [Rozwiązanie problemu znajdziesz tutaj.](../troubleshooting/acre.md)
+
+Q: Co to ACRE2 i TS?.
+
+A: TS - TeamSpeak, taki komunikator. ACRE to plugin do TSa i mod do Army3 w jednym, który ZAWSZE ZNAJDUJE SIĘ W MODLIŚCIE, więc nie musicie się martwić, że tego nie pobraliście. Macie modlistę na misję - macie ACRE. Potrzebujecie jedynie TeamSpeaka, który jest oprogramowaniem darmowym. Zalecam pierwsze odpalenie Army z nasza modlistą przeprowadzić z wyłączonym TSem. ACRE przekopiuje sobie wtedy swoje pliki do katalogów TSa, w innym wypadku nie będzie miało do nich dostępu, bo będą używane przez włączonego TSa. 
+
+TeamSpeaka możecie nawet odpalić w trakcie gry, ale robimy to z reguły przed lub po wejściu do lobby na serwerze. Kiedy już Zeus (Game Master) odpali misję, to automatycznie przerzuci was do odpowiedniego kanału na TSie.
+
+ACRE symuluje komunikację jak w prawdziwości - jeżeli ktoś jest daleko, to usłyszycie go tylko jak krzyknie przy odpowiednio podregulowanej głośności. Są też radia, czyli podstawa łańcucha dowidzenia. Tutaj odsyłam do [poradnika poświęconego ACRE2](../guides/acre), gdzie znajdziecie też informacje na temat obsługi radia. Na serwerze zawsze ktoś wam z tym pomoże jeżeli poprosicie, ale przydaje się wcześniejsza świadomość istnienia takiej mechaniki.
```

Comment:
```suggestion
ACRE symuluje komunikację głosową z pola walki - jeżeli ktoś jest daleko, to usłyszycie go tylko jak krzyknie przy odpowiednio podregulowanej głośności. Są też radia, czyli podstawa łańcucha dowidzenia. Tutaj odsyłam do [poradnika poświęconego ACRE2](../guides/acre), gdzie znajdziecie też informacje na temat obsługi radia. Na serwerze zawsze ktoś wam z tym pomoże jeżeli poprosicie, ale przydaje się wcześniejsza świadomość istnienia takiej mechaniki.
```

Reply:
imo we should either leave the original one or just `komunikację głosową`/`komunikację głosową bliższej realistycznej`

Reply:
> imo we should either leave the original one or just `komunikację głosową`/`komunikację głosową bliższej realistycznej`

@sdsznsk Good point. I edited suggeted change.